local-cli/generator-android/templates/package/MainActivity.java

```diff
@@ -24,7 +24,7 @@ protected String getMainComponentName() {
      */
     @Override
     protected boolean getUseDeveloperSupport() {
-        return BuildConfig.DEBUG;
+        return true;
```

Comment:
@bestander Seems buck has a method to define the `BuildConfig` class - https://buckbuild.com/rule/android_build_config.html